### PR TITLE
DEVEX-2300 `--detailed-job-metrics-collect-default` for `dx update org`

### PR DESCRIPTION
**Pull request by kpjensen**

### CODE REVIEW

File: src/python/dxpy/cli/org.py [hunk 1]
```diff
@@ -252,6 +252,9 @@ def _get_org_update_args(args):
         org_update_inputs["policies"]["jobReuse"] = True
     elif args.disable_job_reuse == True:
         org_update_inputs["policies"]["jobReuse"] = False
+    if args.detailed_job_metrics_collect_default is not None:
+        org_update_inputs["policies"]["detailedJobMetricsCollectDefault"] = True if args.detailed_job_metrics_collect_default == 'true' else False
```
Comment (wormsik) [inline src/python/dxpy/cli/org.py:256]:
```suggestion
        org_update_inputs["policies"]["detailedJobMetricsCollectDefault"] = args.detailed_job_metrics_collect_default == 'true'
```

File: src/python/dxpy/scripts/dx.py [hunk 2]
```diff
@@ -5042,6 +5042,7 @@ def positive_integer(value):
 parser_update_org.add_argument('--member-list-visibility', help='New org membership level that is required to be able to view the membership level and/or permissions of any other member in the specified org (corresponds to the memberListVisibility org policy)', choices=['ADMIN', 'MEMBER', 'PUBLIC'])
 parser_update_org.add_argument('--project-transfer-ability', help='New org membership level that is required to be able to change the billing account of a project that is billed to the specified org, to some other entity (corresponds to the restrictProjectTransfer org policy)', choices=['ADMIN', 'MEMBER'])
 parser_update_org.add_argument('--saml-idp', help='New SAML identity provider')
+parser_update_org.add_argument('--detailed-job-metrics-collect-default', choices=["true", "false"], help="If set to true, jobs launched in the projects billed to this org will collect detailed job metrics by default")
```
Comment (wormsik) [inline src/python/dxpy/scripts/dx.py:5045]:
```suggestion
parser_update_org.add_argument('--detailed-job-metrics-collect-default', choices=['true', 'false'], help='If set to true, jobs launched in the projects billed to this org will collect detailed job metrics by default')
```

**Review by wormsik — APPROVED** [review]
just a minor cosmetic suggestions...feel free to reject...

otherwise LGTM 👍

File: src/python/dxpy/scripts/dx.py [hunk 3]
```diff
@@ -5042,6 +5042,7 @@ def positive_integer(value):
 parser_update_org.add_argument('--member-list-visibility', help='New org membership level that is required to be able to view the membership level and/or permissions of any other member in the specified org (corresponds to the memberListVisibility org policy)', choices=['ADMIN', 'MEMBER', 'PUBLIC'])
 parser_update_org.add_argument('--project-transfer-ability', help='New org membership level that is required to be able to change the billing account of a project that is billed to the specified org, to some other entity (corresponds to the restrictProjectTransfer org policy)', choices=['ADMIN', 'MEMBER'])
 parser_update_org.add_argument('--saml-idp', help='New SAML identity provider')
+parser_update_org.add_argument('--detailed-job-metrics-collect-default', choices=['true', 'false'], help='If set to true, jobs launched in the projects billed to this org will collect detailed job metrics by default')
```
Comment (emiloslavsky) [inline src/python/dxpy/scripts/dx.py:5045]: LGTM

**Pull request merged**